cmd/limactl/factory-reset.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	networks "github.com/lima-vm/lima/pkg/networks/reconcile"
+	"github.com/lima-vm/lima/pkg/start"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newFactoryResetCommand() *cobra.Command {
+	var resetCommand = &cobra.Command{
+		Use:               "factory-reset INSTANCE",
+		Short:             "Factory reset an instance of Lima",
+		Args:              cobra.MaximumNArgs(1),
+		RunE:              factoryResetAction,
+		ValidArgsFunction: factoryResetBashComplete,
+	}
+	return resetCommand
+}
+
+func factoryResetAction(cmd *cobra.Command, args []string) error {
+	instName := DefaultInstanceName
+	if len(args) > 0 {
+		instName = args[0]
+	}
+
+	inst, err := store.Inspect(instName)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			logrus.Infof("Instance %q not found", instName)
+			return nil
+		}
+		return err
+	}
+
+	stopInstanceForcibly(inst)
+
+	fi, err := os.ReadDir(inst.Dir)
+	if err != nil {
+		return err
+	}
+	for _, f := range fi {
+		path := filepath.Join(inst.Dir, f.Name())
+		if !strings.HasSuffix(path, ".yaml") && !strings.HasSuffix(path, ".yml") {
+			logrus.Infof("Removing %q", path)
+			if err := os.Remove(path); err != nil {
+				logrus.Error(err)
+			}
+		}
+	}
+	logrus.Infof("Instance %q has been factory reset", instName)
```

Comment:
We can just exit here. No need to start, as factory-reset would be very rarely used command.

Reply:
SGTM.